.github/workflows/release_gh.yml

```diff
@@ -17,3 +17,4 @@ jobs:
       with:
         github-token: ${{ secrets.GITHUB_TOKEN }}
         upload-url: ${{ github.event.release.upload_url }}
+        package-prefix: adafruit_httpserver
```

Comment:
This is really only used for unconventionally named libraries like `asyncio` which  use import names that don't begin with `adafruit_`.  Forcing it like this achieves the same thing down the line as leaving it out, hence how the mpy files are still gathered correctly as you mentioned - so there's no need for it :)

Reply:
removed this one with the latest commit.